packages/config/src/index.ts

```diff
@@ -238,6 +239,9 @@ export default async (
   pnpmConfig.sharedWorkspaceLockfile = typeof pnpmConfig['sharedWorkspaceLockfile'] === 'undefined'
     ? pnpmConfig.sharedWorkspaceShrinkwrap
     : pnpmConfig['sharedWorkspaceLockfile']
+  pnpmConfig.testPattern = typeof npmConfig['test-pattern'] === 'undefined'
```

Comment:
I am not sure checking `test-pattern` is needed. test-pattern is converted to `testPattern` by the config reader.

Reply:
done [here](https://github.com/pnpm/pnpm/pull/3035/commits/6a684097308b4526cdbf5013b4df6c23b650e4bd)